src/main/java/strategy/OperationStrategy.java

```diff
@@ -0,0 +1,14 @@
+package strategy;
+
+import java.util.Map;
+import model.FruitTransaction;
+import strategy.operation.OperationHandler;
+
+public interface OperationStrategy {
+
+    OperationHandler handler(FruitTransaction fruitTransaction);
+
+    Map<String, Integer> getFruitStatistic();
```

Comment:
it has to be a separate service, strategy Object should only return a correct handler impl